lib/routes/routes.dart

```diff
@@ -120,9 +136,9 @@ GoRoute _getBackgroundSoundRoute() {
   );
 }
 
-GoRoute _getWebviewRoute() {
+GoRoute _getWebviewRoute({bool fromRoot = false}) {
   return GoRoute(
-    path: 'webview',
+    path: fromRoot ? WebviewPath : WebviewPath.replaceFirst('/', ''),
```

Comment:
for these `replaceFirst('/', '')` maybe we could have an ext fun that is super simple called

```
extension SanitisePath on String {
  String sanitisePath() {
    return this.replaceFirst('/', '');
  }
}
```

just makes it a bit neater, what do you think?